src/uu/date/src/date.rs

```diff
@@ -265,8 +267,15 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
         for date in dates {
             match date {
                 Ok(date) => {
+                    // TODO - Revisit when chrono 0.5 is released. https://github.com/chronotope/chrono/issues/970
+                    let tz_str = get_timezone().unwrap_or("Etc/UTC".to_string());
+                    let tz: Tz = tz_str.parse().unwrap();
+                    let offset = tz.offset_from_utc_date(&Utc::now().date_naive());
+                    let tz_abbreviation = offset.abbreviation();
                     // GNU `date` uses `%N` for nano seconds, however crate::chrono uses `%f`
-                    let format_string = &format_string.replace("%N", "%f");
+                    let format_string = &format_string
+                        .replace("%N", "%f")
+                        .replace("%Z", tz_abbreviation);
```

Comment:
This is not quite correct, because this means that `%f` should be escaped. Ultimately, we might need to completely customize this. I suppose it's good enough for now though.

Reply:
Actually, this line of code was already there. Cargo fmt has put it in new line 😀. I am interested in solving this though in my next PR.

I didn't get why %f need to be escaped. Did you mean, if there is %f already within the input, it should be escaped?